src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -52,20 +64,44 @@ Error({Kind:ErrorKind.InvalidArgument})
 Blank()
 
 >> Text(ParseJSON("true"))
-Error({Kind:ErrorKind.InvalidArgument})
+"true"
 
 >> Text(ParseJSON("false"))
-Error({Kind:ErrorKind.InvalidArgument})
+"false"
 
 >> Text(ParseJSON("5"))
-Error({Kind:ErrorKind.InvalidArgument})
+"5"
 
 >> Boolean(ParseJSON("true"))
 true
 
 >> Boolean(ParseJSON("false"))
 false
 
+>> Boolean(ParseJSON("""true"""))
+true
+
+>> Boolean(ParseJSON("""false"""))
+false
+
+>> Boolean(ParseJSON("""tRuE"""))
+true
+
+>> Boolean(ParseJSON("""fAlSe"""))
+false
+
+>> Boolean(ParseJSON("""F"""))
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Boolean(ParseJSON("1"))
+true
```

Comment:
Boolean(ParseJSON("123,456")) ? 

Are we doing
Json --> Text --> Number --> Bool?

Reply:
No

Reply:
You mean somethink like Boolean(ParseJSON("""5""")) ?

Because if so, we are not supporting this case.

Reply:
`Boolean(ParseJSON("123,456"))` is an error, because `123,456` is invalid JSON; `Boolean(ParseJSON("""5"""))` is an error, because the only string values that can be converted to bool are "true" and "false" (case insensitive).